a3-live/src/components/TeamInterface.js

```diff
@@ -7,68 +7,88 @@ import FormControl from '@material-ui/core/FormControl';
 import ListItemText from '@material-ui/core/ListItemText';
 import Select from '@material-ui/core/Select';
 import Checkbox from '@material-ui/core/Checkbox';
+import {OutlinedInput} from "@material-ui/core";
+import { makeStyles } from '@material-ui/core/styles';
+import FilledInput from '@material-ui/core/FilledInput';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import Card from "@material-ui/core/Card";
+import CardHeader from "@material-ui/core/CardHeader";
+import CardContent from "@material-ui/core/CardContent";
 
+const useStyles = makeStyles(theme => ({
+    root: {
+        display: 'flex',
+        flexWrap: 'wrap',
+    },
+
+    selectEmpty: {
+        marginTop: theme.spacing(2),
+    },
+    card: {
+        maxWidth: 345,
+    },
+}));
+/*
+const inputLabel = React.useRef(null); // TODO: Lookup in docs
+const [labelWidth, setLabelWidth] = React.useState(0);
+React.useEffect(() => {
+        setLabelWidth(inputLabel.current.offsetWidth);
+    }, []);
+*/
 
 function TeamInterface(props) {
-    const [TeamLead, setTeamLead] = React.useState({});
+    const classes = useStyles();
     const [TeamMembers, setTeamMembers] = React.useState([]);
+    const [TeamLead, setTeamLead] = React.useState(props.TeamLead);
```

Comment:
Here, a state for Projects the team endures may be declared.
```jsx harmony
    const [TeamLead, setTeamLead] = React.useState(props.TeamLead);
    const [Projects, setProjects] = React.useState(props.Projects);
```
However, mind that `props.Projects` passed into the Hook `useState` will contain list of ALL projects in the system. Hence, you would need to filter `props.Projects` using `props.Team.Projects` that belong to the current **Team**:
```suggestion
    const [TeamLead, setTeamLead] = React.useState(props.TeamLead);
    let assignedProjects = props.Team.Projects.map(assigned => props.Projects.find(prj => prj._id === assigned._id) );
    const [Projects, setProjects] = React.useState(assignedProjects);
```